ecep_cms/README.md

```diff
@@ -0,0 +1,7 @@
+# Example
+
+An extension of the [Core CMS](https://github.com/TACC/Core-CMS) project
+
+## Basics
+
+See [Core-CMS-Custom](https://github.com/TACC/Core-CMS-Custom#readme).
```

Comment:
Nit-picking detail here: in a2cps and ecep you use the full URL for the README, but in the other projects you use the relative path form of `../README.md` for the link. Just fyi in case you wanted to align these.

Reply:
```suggestion
See [Core-CMS-Custom](../README.md).
```

Reply:
And `main` branch commit e626d60 for a2cps fix.